src/pathfinding.h

```diff
@@ -0,0 +1,45 @@
+#ifndef PATHFINDING_H
+#define PATHFINDING_H
+
+enum pf_special : int {
+    PF_NORMAL = 0x00,    // Plain boring tile (grass, dirt, floor etc.)
+    PF_SLOW = 0x01,      // Tile with move cost >2
+    PF_WALL = 0x02,      // Unpassable ter/furn/vehicle
+    PF_VEHICLE = 0x04,   // Any vehicle tile (passable or not)
+    PF_FIELD = 0x08,     // Dangerous field
+    PF_TRAP = 0x10,      // Dangerous trap
+    PF_UPDOWN = 0x20,    // Stairs, ramp etc.
+};
+
+constexpr pf_special operator | ( pf_special lhs, pf_special rhs )
```

Comment:
Apparently they aren't. Trying to use them casts the enum down to int and then complains about invalid assignment.